pkg/awsvpc/parser.go

```diff
@@ -291,6 +296,65 @@ func (rc *AWSresourcesContainer) getSGconfig(
 	return nil
 }
 
+func (rc *AWSresourcesContainer) getNACLconfig(
+	res *vpcmodel.MultipleVPCConfigs,
+	skipByVPC map[string]bool,
+) error {
+	naclLists := map[string][]*commonvpc.NACL{} // map from vpc uid to its nacls
+	for i := range rc.NetworkACLsList {
+		nacl := rc.NetworkACLsList[i]
+		if skipByVPC[*nacl.VpcId] {
+			continue
+		}
+		naclAnalyzer, err := commonvpc.NewNACLAnalyzer(NewAWSNACLAnalyzer(nacl))
+		if err != nil {
+			return err
+		}
+		vpcUID := *nacl.VpcId
+		vpc, err := commonvpc.GetVPCObjectByUID(res, vpcUID)
+		if err != nil {
+			return err
+		}
+
+		naclResource := &commonvpc.NACL{
+			VPCResource: vpcmodel.VPCResource{
+				ResourceName: *nacl.NetworkAclId,
+				ResourceUID:  *nacl.NetworkAclId,
+				ResourceType: commonvpc.ResourceTypeNACL,
+				VPCRef:       vpc,
+				Region:       vpc.RegionName(),
+			},
+			Analyzer: naclAnalyzer, Subnets: map[string]*commonvpc.Subnet{}}
+		naclLists[vpcUID] = append(naclLists[vpcUID], naclResource)
+		for _, subnetRef := range nacl.Associations {
```

Comment:
```suggestion
		for i := range nacl.Associations {
			subnetRef := &nacl.Associations[i]
```
When you have a slice of structs, this is the best syntax (avoids copying the structs)